pkg/compare/compare.go

```diff
@@ -523,6 +574,7 @@ type DiffSum struct {
 	DiffOutput         string `json:"DiffOutput"`
 	CorrelatedTemplate string `json:"CorrelatedTemplate"`
 	CRName             string `json:"CRName"`
+	Patch              string
```

Comment:
please remove patch from this PR, here in also in name of variables and other functions :)